apps/mymedicare_cb/models.py

```diff
@@ -61,71 +61,92 @@ def get_and_update_user(
 
     log_dict = {
         "type": "mymedicare_cb:get_and_update_user",
+        "subject": subject,
         "fhir_id": fhir_id,
         "mbi_hash": mbi_hash,
         "hicn_hash": hicn_hash,
         "hash_lookup_type": hash_lookup_type,
         "crosswalk": {},
+        "crosswalk_before": {},
     }
 
+    # Init for types of crosswalk updates.
+    hicn_updated = False
+    mbi_updated = False
+    mbi_updated_from_null = False
```

Comment:
Refactored to use these flags to audit the type of Crosswalk changes in the logs. This is from Nick's review suggestion.